apps/bfd-pipeline/bfd-pipeline-rda-grpc-server/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/ConfigLoader.java

```diff
@@ -0,0 +1,271 @@
+package gov.cms.bfd.pipeline.rda.grpc.sink;
+
+import com.google.common.base.Strings;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.function.Function;
+
+/** Simple class to abstract loading configuration values from multiple sources. */
+public class ConfigLoader {
```

Comment:
I could have sworn you implemented a class like this previously, was it not reusable or am I mistaken?

Reply:
Similar yes but less flexible than this one.  That one wound up being removed during the pipeline refactoring a while back.  I think it's important to have something like this available and I needed it here so I added it.  It's much more flexible than just reading environment variables and removes a ton of boilerplate.  It'd be nice to see it incorporated into the pipeline and/or server configuration.